### DISCUSSION TITLE
How do you edit email templates from the dashboard?

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#2677] [Q&A]
We don't use the CLI for various reasons  and don't want to, and just use the Nhost dashboard. We cannot fix the email templates for auth? The docs (https://docs.nhost.io/guides/auth/email-templates) say, to connect a github repo, but then when we click on the link to connect a github repo, there are no directions. Anyway, why would we connect a github repo, if we are managing everything thru the dashboard. There needs to be a way to modify the email templates in the dashboard, or else we can't use the Auth. Please let us know how to do modify templates without having to set up a CLI locally. And actually, we use postmark for emails, so want to use their templates, but the docs are kind of sparse on this topic and it's not clear what exactly to do. How do we set up the integration with postmark? What exact templates are used and how do we pass the variables?

**Comment by dbarrosop** [comment]
Editing templates in the dashboard isn't supported. From what you are saying, using postmark's native integration is a better solution.

> How do we set up the integration with postmark? 

To configure it got to settings -> SMTP and select "postmark" at the top. Then enter the sender you want to use and your API token in the password.

> What exact templates are used?

From the docs:

In postmark you need to create three templates for each locale with the following alias:
- $locale.email-change
- $locale.email-verify
- $locale.password-reset

For instance:

- en.email-change
- en.email-verify
- en.password-reset
- se.email-change
- se.email-verify
- se.password-reset

> how do we pass the variables?

You don't, you can use the variables mentioned in the documentation. The variables will automatically be sent by the Auth service when needed. In case of doubt you can start with the templates we have in our opensource repo and modify from there (keep in mind the templates we have in our repo are not for postmark so you may need to adapt accordingly):

https://github.com/nhost/hasura-auth/tree/main/email-templates/en

**Reply by osseonews** [reply]
Thanks. Didn't see the setting for postmark in settings before. Will test. Also, what about the other templates for postmark, like for signin-passwordless, is the pattern the same for setting up those templates in postmark? do we do, en.signin-passwordless as the template in postmark?

**Reply by dbarrosop** [reply]
> en.signin-passwordless as the template in postmark?

yes, exactly. I will update our docs next week, this one was left out by mistake
